packages/core/src/models/UISchema/accessors/object.ts

```diff
@@ -27,8 +27,10 @@ export interface ObjectAccessorInterface extends BaseAccessorInterface {
     getGroupPosition(groupName: string): number;
     isPropertyInGroup(groupName: string, prop: string): boolean;
 
-    setOrder(iterator: (order: string[]) => string[]): this;
-    getOrder(): string[] | undefined;
+    setOrder(
+        iterator: (order: (string | string[])[]) => (string | string[])[]
+    ): this;
```

Comment:
```ts
setOrder(
        iterator: (order: Array<string | string[]>) => Array<string | string[]>
    ): this;
```
might be readable a little better

Reply:
<img width="658" alt="image" src="https://user-images.githubusercontent.com/122360/207845949-2f79cef0-0dd9-4637-a42b-6c387f38d708.png">
🤷🏻‍♂️